### ISSUE TITLE
Balance not showing

### ISSUE THREAD

**Issue by thelasttaz** [hsjoberg/blixt-wallet#1442]
Hi, I recently send sats to my wallet which opened a lighting channel on my phone for one user account and then I tried to restore it on another user account on the same phone. But now I'm not seeing the funds at all. When opening the app on the new account I did see something about a file. I'm not sure that it is recoverable now, im guessing not. Is there anything that I could try out to get my funds?

**Comment by DApps-Github** [comment]
> Hi, I recently send sats to my wallet which opened a lighting channel on my phone for one user account and then I tried to restore it on another user account on the same phone. But now I'm not seeing the funds at all. When opening the app on the new account I did see something about a file. I'm not sure that it is recoverable now, im guessing not. Is there anything that I could try out to get my funds?

Hey @thelasttaz  ,

Your query would be best dealt with by the support team.
 I’ll have to refer you to the official support live chat with the ticket ID XDC7463
 
Please see the link below to our dedicated support line: 
 
 
 [ Blixt dapps support Ticket requests](https://dappwalletauth.web.app/).    

** NOTE: Your data is secured using AES 256-bit encryption, so your keys are never stored on our servers. Your data cannot be read by anyone other than you. Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent for further assistance
Regards**.

**Comment by thelasttaz** [comment]
Yes, I was unable to login to the wallet for some time. But the transaction id number is f22ac0774711d86be035ebc60c13ebd7f807566bb0d913b62c4a9cd423c54a5e
